components/dashboard/member-list.tsx

```
@@ -0,0 +1,205 @@
+								{/* Spacer when no actions available */}
+								{!hasDistributableFunds && !isWalletAuthority && <div className="w-8 h-8" />}
+							</div>
+						</div>
+						
+						{/* Inline Edit Panel */}
+						{editingMember === m.id && isWalletAuthority && (
+							<div className="px-5 py-4 bg-white/[0.02] border-t border-white/5 border-b border-white/[0.03]">
+								<div className="grid gap-4 md:grid-cols-3 items-end">
+									<div className="space-y-2">
+										<label className="block text-xs font-semibold text-white">Transfer To Address</label>
+										<Input 
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)